src/utils/dates.js

```diff
@@ -1,4 +1,4 @@
-const ONE_DAY_IN_MILLISECONDS = 86400000;
+export const ONE_DAY_IN_MILLISECONDS = 86400000;
```

Comment:
Nice use of uppercase text for this immutable global variable! I needed to read about the difference between this and const to understand it better.

Reply:
This variable was declared in the codebase before we all got started, and when we saw it, we we're like yoink! gotta use this elsewhere haha